test/integration/conftest.py

```diff
@@ -14,27 +14,32 @@
 from test.utils.revert_language_settings import revert_language_settings
 from test.utils.db_utils import create_schema, open_schema
 
-
-SLC_NAME = "template-Exasol-all-python-3.10_release.tar.gz"
-
-SLC_URL_FORMATTER = ("https://github.com/exasol/script-languages-release/releases/"
-                     "download/{version}/") + SLC_NAME
-
 VERSION = "8.0.0"
 
 TEST_SCHEMA = "PEC_DEPLOYER_TESTS"
 TEST_LANGUAGE_ALIAS = "PYTHON3_PEC_TESTS"
 
 
+@pytest.fixture(scope='session')
+def container_name() -> str:
+    return "template-Exasol-all-python-3.10_release.tar.gz"
+
+
+@pytest.fixture(scope='session')
+def container_url_formatter(container_name) -> str:
+    return ("https://github.com/exasol/script-languages-release/releases/"
+            "download/{version}/") + container_name
```

Comment:
```suggestion
    return (
           "https://github.com/exasol/script-languages-release/releases/"
            "download/{version}/"
            container_name
    )
```

Reply:
Hmmm, can we do this?

Reply:
If it is all strings should work just fine, try it out in an interactive session.

```
expected = "abcdefg"
val = (
  "ab"
  "cdef"
  "g"
)

assert expected == val